### ISSUE TITLE
Node limiting should use NES

### ISSUE THREAD

**Issue by mikekucera** [cytoscape/enrichment-map-webapp#116]
**Describe the bug**

Currently we limit the number of nodes in the network by sorting by q-value then taking the top 50. Since we changed to showing the NES value on the network we should be using the magnitude of the NES value to limit the nodes.

https://github.com/cytoscape/enrichment-map-webapp/blob/main/src/client/components/network-editor/index.js#L71

Also, could this be done on the server, to avoid transferring a bunch of unneeded data to the client? If so then edges between filtered-out nodes should also be filtered out before sending to the client.

**Comment by maxkfranz** [comment]
> Also, could this be done on the server, to avoid transferring a bunch of unneeded data to the client? If so then edges between filtered-out nodes should also be filtered out before sending to the client.

What's the file size (unfiltered versus filtered)?

**Comment by maxkfranz** [comment]
> Also, could this be done on the server [...]?

Load it into a cytoscape instance on the server, just the same as the client, and then you can remove nodes (which automatically removes attached edges).  

The code for the filter itself could probably just be copy-pasted over from the client to the server.  Cytoscape was designed so you can use it wherever you want.

**Comment by mikekucera** [comment]
Not sure the exact size of the result body data. But for the GSE129943_rsem_counts_2016 network it originally has 394 nodes and 285 edges, after filtering that becomes 50 nodes and 26 edges. So approximately 11% of the elements remain after filtering.

**Comment by maxkfranz** [comment]
For the scale of <1000 elements, it's probably not going to make a significant effect on performance in the short term.  On the other hand, it may not be a complicated change to make -- so six of one.

Performance is a good topic to consider for future iterations, generally.  I've created [an issue](https://github.com/cytoscape/enrichment-map-webapp/issues/117) to track discussion topics for when we reach the end of the milestone.  I've added performance to the list.  Feel free to edit and add new items.
